http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/RuntimeRoutes.scala

```diff
@@ -547,6 +547,7 @@ object RuntimeRoutes {
       ("jupyterStartUserScriptUri", x.startUserScriptUri.asJson),
       ("errors", x.errors.asJson),
       ("userJupyterExtensionConfig", x.userJupyterExtensionConfig.asJson),
+      ("autopause", x.autopause.asJson),
```

Comment:
so I think we can probably remove `autopause` from `GetRuntimeResponse` as well, and just adding this field in the encoder to make UI easier.

you can have
```
val autoPauseEnabled = x.autopauseThreshold != autoPauseOffValue

Json.obj(
...
("autopause", autoPauseEnabled.asJson)
...
```

Reply:
Where is the encoder? Still getting familiar with the codebase